db/tunnel.go

```diff
@@ -86,7 +86,8 @@ func Tunnel(opts TunnelOpts) error {
 	}
 
 	defer sock.Close()
-	fmt.Printf("You can access your database on '%v'\n", sock.Addr())
+	fmt.Println("You can access your database on:")
```

Comment:
Don't you want to display this on stderr also?

Reply:
Ugh, I used grep to test what goes in stdout but I did `grep ":"` and that line ends with one. <facepalm>

Reply:
*facepalm*